tools/pythonpkg/lib_name.hpp

```diff
@@ -0,0 +1 @@
+#define NAME duckdb
```

Comment:
Perhaps this can be passed with a define (-D) so the file doesn’t need to exist? The main lib can have a line eg

#ifndef DUCKDB_PYTHON_LIB_NAME
#define DUCKDB_PYTHON_LIB_NAME “duckdb”
#endif